llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -1548,10 +1605,36 @@ DbgEntity *DwarfDebug::createConcreteEntity(DwarfCompileUnit &TheCU,
   return ConcreteEntities.back().get();
 }
 
+void DwarfDebug::populateDependentTypeMap() {
+  for (const auto &I : DbgValues) {
+    InlinedEntity IV = I.first;
+    if (I.second.empty())
+      continue;
+    if (const DIVariable *DIV = dyn_cast<DIVariable>(IV.first)) {
+      if (const DIStringType *ST = dyn_cast<DIStringType>(
+              static_cast<const Metadata *>(DIV->getType())))
+        if (const DIVariable *LV = ST->getStringLength())
+          VariableInDependentType[LV] = ST;
+
+      if (const DIFortranArrayType *AT = dyn_cast<DIFortranArrayType>(
+              static_cast<const Metadata *>(DIV->getType())))
+        for (const DINode *S : AT->getElements())
+          if (const DIFortranSubrange *FS = dyn_cast<DIFortranSubrange>(S)) {
+            if (const DIVariable *LBV = FS->getLowerBound())
+              VariableInDependentType[LBV] = AT;
+            if (const DIVariable *UBV = FS->getUpperBound())
+              VariableInDependentType[UBV] = AT;
+        }
```

Comment:
There is a misaligned brace on the bottom here (aligned with `for`, should be aligned with `if`). The top `for` and the `if` should both have braces as they have multiline bodies.

```suggestion
      if (const DIFortranArrayType *AT = dyn_cast<DIFortranArrayType>(
              static_cast<const Metadata *>(DIV->getType()))) {
        for (const DINode *S : AT->getElements()) {
          if (const DIFortranSubrange *FS = dyn_cast<DIFortranSubrange>(S)) {
            if (const DIVariable *LBV = FS->getLowerBound())
              VariableInDependentType[LBV] = AT;
            if (const DIVariable *UBV = FS->getUpperBound())
              VariableInDependentType[UBV] = AT;
          }
        }
      }
```

Reply:
Fixed.